packages/vc-export/src/KiltRevocationStatusV1.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) 2018-2023, BOTLabs GmbH.
+ *
+ * This source code is licensed under the BSD 4-Clause "Original" license
+ * found in the LICENSE file in the root directory of this source tree.
+ */
+
+import { u8aToHex, u8aToU8a } from '@polkadot/util'
+import { base58Decode, base58Encode } from '@polkadot/util-crypto'
+import type { ApiPromise } from '@polkadot/api'
+import type { U8aLike } from '@polkadot/util/types'
+
+import { ConfigService } from '@kiltprotocol/config'
+import { Attestation, CType, SDKErrors } from '@kiltprotocol/core'
+import type { Caip2ChainId } from '@kiltprotocol/types'
+
+import { Caip2 } from './CAIP/index.js'
+import { KILT_REVOCATION_STATUS_V1_TYPE } from './constants.js'
+import {
+  assertMatchingConnection,
+  getDelegationNodeIdForCredential,
+} from './common.js'
+import { CredentialMalformedError } from './errors.js'
+import type { KiltRevocationStatusV1, VerifiableCredential } from './types.js'
+
+/**
+ * Check attestation and revocation status of a credential at the latest block available.
+ *
+ * @param credential The KiltCredentialV1 to which the status method is linked to.
+ * @param opts Additional parameters.
+ * @param opts.api An optional polkadot-js/api instance connected to the blockchain network on which the credential is anchored.
+ * If not given this function will try to retrieve a cached connection from the [[ConfigService]].
+ */
+export async function checkStatus(
```

Comment:
Perhaps simply `check`?

Reply:
Check... what? A proof? A credential? Where is the difference to `verify`? I'd rather go more specific: `checkRevocationStatus`?

Reply:
Oh now I see it, the namespace should explain it... yeah I guess that works then?

Reply:
62ee6471494952e15a63b068f6f1ee48dfd802bc